CHANGELOG.md

```diff
@@ -5,7 +5,50 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.1.1] - 2020-02-07
+## [0.4] - 2021-06-10
+### Fixed
+- More accurately calculates expected size of plots.
+- Archival requires only minimal extra space on target drive.
+  The required space is based on the size of the actual plot to be transferred.
+  Previously a 20% (~20GB) margin was required relative to a rough approximation of plot size.
+- Identify more cases of chia plotting processes such as on NixOS.
+- Avoid some more `NoSuchProcess` and `AccessDenied` errors when identifying plotting processes.
+- Avoid crashing when parsing plotting process logs fails to decode due to `UnicodeDecodeError`.
+- Avoid crashing when a tmp file is removed while we are checking a job's tmp usage.
+- Windows is not yet supported, but plot and archive processes are now launched to be independent of the plotman process on Windows as it already was on Linux.
+### Added
+- Configuration file is versioned.
+  The config for previous plotman versions has been retroactively defined to be version 0
+  The new version is 1.
+  An error will be raised when launching plotman which includes a link to the wiki.
```

Comment:
Something missing here?  Do you mean if an error is raised when launching plotman, a wiki link will be included?

Reply:
I reworded it and mucked it up in the process.  Thanks.

```suggestion
  An error will be raised when you launch plotman with a configuration file whose version does not match the expected configuration version.
  That error will include a link to the wiki to help understand the needed changes.
```